engine/common/requester/engine.go

```diff
@@ -404,6 +409,9 @@ func (e *Engine) process(originID flow.Identifier, message interface{}) error {
 }
 
 func (e *Engine) onEntityResponse(originID flow.Identifier, res *messages.EntityResponse) error {
+	log := e.log.With().Str("origin_id", originID.String()).Uint64("nonce", res.Nonce).Logger()
```

Comment:
```suggestion
	lg := e.log.With().Str("origin_id", originID.String()).Uint64("nonce", res.Nonce).Logger()
```
to avoid any variable and package name collision.

Reply:
https://github.com/onflow/flow-go/pull/3270/commits/3eb42cb65e1f6f08f51edcfdd93c4e1d466c8608